content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -40,6 +40,49 @@ The following configuration options behave consistently across the latest versio
 **Supported Input**: Boolean <br>
 **Description**: Enables or disables sending traces from the application.
 
+`DD_TRACE_SAMPLE_RATE`
+: **Default**: `-1`. If unset, the tracer defers to the Datadog Agent to control sample rate. <br>
+**Supported Input**: A float between 0.0 and 1.0, inclusive.  <br>
+**Caveats**: This variable is deprecated in favor of `DD_TRACE_SAMPLING_RULES`, which provides more flexible and granular sampling control.  <br>
+**Description**: Controls the trace ingestion sample rate between the Datadog Agent and the backend. Must be a float between 0.0 and 1.0, where 1.0 means all traces are sent to the backend and 0.0 means none are sent. This setting applies globally to all traces and does not support per-service or per-operation targeting.
+
+`DD_TRACE_SAMPLING_RULES`
+: **Default**: `null`. If unset or no rules match, the tracer defers to the Datadog Agent to dynamically adjust sample rate across traces.  <br>
+**Supported Input**: A JSON array of objects. <br>
```

Comment:
Wondering if we should change the wording here to `A JSON array of SamplingRule.` and define that below along with the types of `service`, `name`, `resource`, `tags`, and `max_per_second`. Not sure if that would fit on this page though

Maybe this should just link to the sampling page with the specifics

@dussault-antoine wdyt?

Reply:
Yep, I would suggest to simply link to the ingestion control page: https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_mechanisms/?tab=java#in-tracing-libraries-user-defined-rules

Reply:
I'd suggest a different link on line 51, something like the below (if possible)

```suggestion
**Supported Input**: A JSON array of [user-defined rules](https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_mechanisms/?tab=java#in-tracing-libraries-user-defined-rules). <br>
```